--- PKGBUILD	2024-10-20 18:20:45.038697386 -0300
+++ PKGBUILD	2024-10-20 18:21:41.281754526 -0300
@@ -97,14 +97,14 @@
 #  43. Generic-x86-64-v4 (GENERIC_CPU4)
 #  44. Intel-Native optimizations autodetected by the compiler (MNATIVE_INTEL)
 #  45. AMD-Native optimizations autodetected by the compiler (MNATIVE_AMD)
-: "${_subarch:=""}"
+: "${_subarch:="24"}"
 
 # Enable compilation with LLVM
 # Be warned, this is largely untested by me (JeremyStarTM). It *should* work,
 # but if it doesn't, write a comment and I'll fix it.
 # 
 # Set to anything but null to activate.
-: "${_use_llvm_lto:=""}"
+: "${_use_llvm_lto:="1"}"
 
 # Debug options
 # This allows you to enable or disable debug options.
@@ -306,7 +306,7 @@
 
 build() {
     cd "${_src_linux}" || exit 1
-    make ${BUILD_FLAGS[*]} all
+    make ${BUILD_FLAGS[*]} -j$(nproc) all
 }
 
 _package() {
